### ISSUE TITLE
Required indentation for bullet lists looks wrong and is inconsistent with numbered lists

### ISSUE THREAD

**Issue by foolip** [tabatkins/bikeshed#1694]
A minimal repro case:
```
<pre class=metadata>
Title: bug repro
Shortname: bug-repro
Status: LD
Editor:
Abstract:
</pre>

This is fine:
 1. Some lines are short
 2. But sometimes lines are long
    and they wrap

But this is an error:
 - Some lines are short
 - But sometimes lines are long
   and they wrap
```

The problem is on the last line. The error message is "Line 17 isn't indented enough (needs 1 indent) to be valid Markdown"

It's already indented 3 spaces, and indenting it 4 spaces does fix the problem. However, that makes the indentation inconsistent.

**Comment by foolip** [comment]
Pasting the same into https://spec.commonmark.org/dingus/ parses correctly, so I think this is a bug in Bikeshed.

**Comment by tabatkins** [comment]
Ah, yeah, that's currently a diversion from CommonMark - I do require you to indent per your recorded `Indent` metadata on subsequent lines of a list item.

The indent is completely consistent between the two, it just so happens that the amount of indent you're using for numbered lists is 4, which matches the default `Indent`.

**Comment by foolip** [comment]
Wait, so the amount of required indentation has nothing to do where the first character of the first list item is? What does CommonMark do?

**Comment by tabatkins** [comment]
Correct. CommonMark sets the required indent to be "however many characters it is from the start of the line to the start of the text after the bullet". So that's four in your numbered-list examples there (but would be 5 once you got up to `10.`), and 3 for your bulleted list.

(Hand-wavey, it's "same column as the text starts in the first line", but that's not technically true since if it's nested in another block the lines could start on different columns.)

Bikeshed's currently being much simpler and just saying "one indent inward", however you define your indents.  Note that Tabs are *always* 1 indent-worth no matter what; I recommend using them. ^_^

**Comment by jyasskin** [comment]
Is this just a Wontfix, or do you expect to align this with CommonMark eventually?

**Comment by tabatkins** [comment]
Hm, I think I'm actually not quite sure. We'll leave it open for now, I'll reevaluate later this year after the parser rewrite is more mature.
